cime_config/buildnml

```diff
@@ -237,6 +237,9 @@ set OALKSCEN = "''"
 set OALKFILE = "''"
 set WITH_DMSPH = .false.
 set PI_PH_FILE = "''"
+set L_SED_POR = .false.
```

Comment:
Maybe a more suggestive name would be preferable? For example L_SEDPOR_2D to indicate that this activates a 2d-spatially variable sediment porosity?

Reply:
Done